x-pack/plugins/uptime/public/components/monitor/synthetics/step_detail/waterfall/data_formatting.ts

```diff
@@ -91,6 +91,22 @@ export const getSeriesAndDomain = (items: NetworkItems) => {
         currentOffset = y;
       }
     });
+
+    /* if no specific timing values are found, use the total time */
+    if (!acc.find((entry) => entry.x === index)) {
+      acc.push({
+        x: index,
+        y0: currentOffset,
+        y: currentOffset + item.timings.total,
```

Comment:
so total will always be there?

Reply:
Vignesh believes that it will be available for static files.

I've added additional logic to account for scenarios in which total is unavailable, as well as scenarios in which the entire timings object is unavailable.

I noticed that this is the case for when the connection is refused. All items, including the total, are listed as `-1`. With my new logic, this is what the behavior would be. There is no hover state. 

This is when there is nothing up and running on localhost:8000
<img width="1402" alt="Screen Shot 2021-01-07 at 12 44 26 PM" src="https://user-images.githubusercontent.com/11356435/103938152-cd832400-50f7-11eb-89df-9a42668da56c.png">

Reply:
Correction: I'm able to remove the tooltip but not the background hover state, which seems to come bundled with passing a custom tooltip to the BarSeries from Elastic Charts. I've yet to discover a way to either use a custom tooltip or specify the tooltip as none on a per datum basis. I can continue the conversation in the elastic charts chat on Monday. 

<img width="1103" alt="Screen Shot 2021-01-10 at 9 23 53 PM" src="https://user-images.githubusercontent.com/11356435/104142687-97a99e00-538a-11eb-8c35-a8c7a0a229e0.png">
<img width="1092" alt="Screen Shot 2021-01-10 at 8 55 05 PM" src="https://user-images.githubusercontent.com/11356435/104142688-98423480-538a-11eb-9068-32fd21703b40.png">

Reply:
Total would always be there for all requests. We need to just use them as fallback when we don't have timings for other parts of the request.